raiden_libs/test/mocks/client.py

```diff
@@ -258,6 +259,21 @@ def get_fee_info(self, partner_address, **kwargs) -> FeeInfo:
         fi.signature = encode_hex(self.sign_data(fi.serialize_bin(), self.privkey))
         return fi
 
+    def request_paths(self, target_address, **kwargs) -> RequestPaths:
+        """Generate a RequestPaths for pathfinding-service .
+        Parameters:
+            token_network_address 	address
```

Comment:
The parameters described here don't match the function signature. There should also be a space after the parameter name in the docstring. See https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html